Hmm, in fact updating Qbs did solve the problem it seems, and now it is running into problems compiling qaseprite due to the newer CMake:
```
CMake Error at aseprite/third_party/libpng/CMakeLists.txt:33 (cmake_minimum_required):
  Compatibility with CMake < 3.5 has been removed from CMake.

  Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
  to tell CMake that the project requires at least <min> but has been updated
  to work with policies introduced by <max> or earlier.

  Or, add -DCMAKE_POLICY_VERSION_MINIMUM=3.5 to try configuring anyway.
```
But this can be solved by updating qaseprite, which I'll do next.

--- a/org.mapeditor.Tiled.json
+++ b/org.mapeditor.Tiled.json
@@ -75,7 +75,6 @@
                 {
                     "type": "git",
                     "url": "https://github.com/mapeditor/tiled.git",
-                    "branch": "1.11",
                     "commit": "b88da92804132eefb2e1dbb595a45bedc2f842e8",
                     "x-checker-data": {
                         "type": "anitya",

--- a/org.mapeditor.Tiled.json
+++ b/org.mapeditor.Tiled.json
@@ -75,7 +75,8 @@
                 {
                     "type": "git",
                     "url": "https://github.com/mapeditor/tiled.git",
-                    "commit": "93fb1e926556794ef3298f8c2b0916105dec47c2",
+                    "branch": "1.11",
+                    "commit": "b88da92804132eefb2e1dbb595a45bedc2f842e8",
                     "x-checker-data": {
                         "type": "anitya",
                         "project-id": 7298,

--- a/org.mapeditor.Tiled.json
+++ b/org.mapeditor.Tiled.json
@@ -28,8 +28,8 @@
             "sources": [
                 {
                     "type": "archive",
-                    "url": "https://github.com/qbs/qbs/archive/refs/tags/v2.6.1.tar.gz",
-                    "sha256": "bca391dd01168e075305baddcef7d4cf0a00b8543ee70b43942817db72d64904"
+                    "url": "https://github.com/qbs/qbs/archive/refs/tags/v3.1.2.tar.gz",
+                    "sha256": "33a981d51615a0115ecf8ec17b237c821cc403c613f4bc8623207d63a9183dc0"
                 }
             ],
             "cleanup": [

